staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package request
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	minimumSeats                = 1
+	maximumSeats                = 10
+	objectsPerSeat              = 100.0
+	watchesPerSeat              = 10.0
+	enableMutatingWorkEstimator = true
```

Comment:
I think these names should include the word "default", since these are only default values.